node/src/services/builder.rs

```diff
@@ -0,0 +1,166 @@
+use async_channel::Receiver;
+use sc_network::{config::IncomingRequest, ProtocolName};
+use sc_service::RpcHandlers;
+use shc_common::types::HasherOutT;
+use sp_keystore::KeystorePtr;
+use sp_trie::TrieLayout;
+use std::{marker::PhantomData, sync::Arc};
+use tokio::sync::RwLock;
+
+use file_manager::{in_memory::InMemoryFileStorage, traits::FileStorage};
+use forest_manager::{
+    in_memory::InMemoryForestStorage, rocksdb::RocksDBForestStorage, traits::ForestStorage,
+};
+use storage_hub_infra::actor::{ActorHandle, TaskSpawner};
+
+use crate::service::{ParachainClient, ParachainNetworkService};
+
+use super::{
+    blockchain::{spawn_blockchain_service, BlockchainService},
+    file_transfer::{spawn_file_transfer_service, FileTransferService},
+    handler::StorageHubHandler,
+};
+
+/// Builds the `StorageHubHandler` by adding each component separately.
```

Comment:
```suggestion
/// Builds the [`StorageHubHandler`] by adding each component separately.
```

I love linking in docs...